tasks/notify.py

```diff
@@ -529,6 +539,9 @@ def save_patch_totals(self, comparison: ComparisonProxy) -> None:
             return
         head_commit = comparison.head.commit
         db_session = head_commit.get_db_session()
+        if db_session is None:
```

Comment:
under which circumstances does this ever happen?

Reply:
I'm not sure. I don't think it should happen, but my type hinter tells me this the result of `Commit.get_db_session()` is `Any | None`. 

Also we have issues [WORKER-N5V](https://codecov.sentry.io/issues/5448418739/?project=4165036&query=%27NoneType%27&referrer=issue-stream&statsPeriod=7d&stream_index=3&utc=true) and [WORKER-NH1](https://codecov.sentry.io/issues/5620843269/?project=4165036&query=%27NoneType%27&referrer=issue-stream&statsPeriod=7d&stream_index=4&utc=true) - not in this task exactly - that show `dbsession` being `None`